### PR TITLE
feat(EMS-2474): No PDF - Your buyer - Buyer financial information page setup

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:
This PR sets up the buyer information page

## Resolution :heavy_check_mark:
* Added buyer information page controllers and routes and content strings
* Added unit tests
* Added e2e test

### CODE REVIEW

File: e2e-tests/commands/insurance/your-buyer/complete-and-submit-buyer-financial-information-form.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * completeAndBuyerFinancialInformationForm
+ * Completes and submits the "buyer financial information" form.
+ * @param {Object} Object with flags on how to complete the form.
+ * - holdBuyerFinancialAccounts: Should submit "yes" to "buyer financial information" radio. Defaults to "no".
+ */
+const completeAndBuyerFinancialInformationForm = ({ holdBuyerFinancialAccounts = false }) => {
```
Comment (ttbarnes) [inline e2e-tests/commands/insurance/your-buyer/complete-and-submit-buyer-financial-information-form.js:7]: `completeAnd`.. ?

File: e2e-tests/pages/insurance/your-buyer/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 import companyOrOrganisationPage from './companyOrOrganisation';
+import buyerFinancialInformationPage from './buyerFinancialAccounts';
 
-export { companyOrOrganisationPage };
+export { companyOrOrganisationPage, buyerFinancialInformationPage };
```
Comment (ttbarnes) [inline e2e-tests/pages/insurance/your-buyer/index.js:4]: `buyerFinancialInformation` vs `buyerFinancialAccounts` ?

Comment (Zainzzkk) [reply e2e-tests/pages/insurance/your-buyer/index.js:4]: the page/url is buyer-financial-information but the field is buyerFinancialAccounts.  Should maybe keep it as buyerFinancialInformation?

File: src/ui/server/controllers/insurance/your-buyer/buyer-financial-information/index.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+const { BUYER_FINANCIAL_ACCOUNTS } = YOUR_BUYER_FIELD_IDS;
+
+export const FIELD_IDS = [BUYER_FINANCIAL_ACCOUNTS];
+
+export const TEMPLATE = SHARED_PAGES.SINGLE_RADIO;
+
+export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER.BUYER_FINANCIAL_INFORMATION;
+
+export const pageVariables = (referenceNumber: number) => ({
```
Comment (ttbarnes) [inline src/ui/server/controllers/insurance/your-buyer/buyer-financial-information/index.ts:38]: Documentation 😛

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**